playwright/tests/comments-expanded.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { breakpoints } from '@guardian/source-foundations';
+import { expect, test } from '@playwright/test';
+import { articles } from '../fixtures/pages';
+import type { GuPage } from '../fixtures/pages/Page';
+import { cmpAcceptAll } from '../lib/cmp';
+import { loadPage } from '../lib/load-page';
+import { waitForIsland, waitForSlot } from '../lib/util';
+
+const { path } = articles[6] as unknown as GuPage;
```

Comment:
Is there only one page we use for this test? In which case we could create the path here rather than adding to `articles.ts`, which I think also increases the time taken for other tests as they would now run against that article too.

Reply:
Good shout, we only need it for this test. I'll define it just for this file